neqo-bin/src/client/mod.rs

```diff
@@ -508,8 +508,8 @@ fn qlog_new(args: &Args, hostname: &str, cid: &ConnectionId) -> Res<NeqoQlog> {
     NeqoQlog::enabled_with_file(
         qlog_dir,
         Role::Client,
-        Some("Example qlog".to_string()),
-        Some("Example qlog description".to_string()),
+        Some("Neqo client qlog".to_string()),
+        Some("Neqo client qlog".to_string()),
         format!("{hostname}-{cid}"),
```

Comment:
```suggestion
        format!("client-{hostname}-{cid}"),
```

I suggest doing the same for the client, i.e. prefixing it with `client-`.

Reply:
Makes sense, I added it in 56cd6cf!